testing/test_sum.py

```diff
@@ -0,0 +1,18 @@
+from ie_pandas import dataframe
```

Comment:
It's fine if you have one file per test, although normally they are grouped in some way